aries_cloudagent/resolver/base.py

```diff
@@ -59,11 +59,11 @@ def native(self):
         return self.type == ResolverType.NATIVE
 
     @property
-    @abstractmethod
     def supported_methods(self) -> Sequence[str]:
         """Return list of DID methods supported by this resolver."""
+        raise NotImplementedError()
```

Comment:
Instead of a `supported_methods` method, we should swap this out for a `supported_did_regex` class variable that is `None` or empty in the base class and must be overridden by subclasses for the default supports functionality.